data/yaml/name_variants.yaml

```diff
@@ -5482,19 +5482,12 @@
 - canonical: {first: Yang, last: Liu}
   comment: Microsoft Cognitive Services Research
   id: yang-liu-microsoft
-- canonical: {first: Yang, last: Liu}
-  comment: May refer to several people
-  id: yang-liu
-  similar: [yang-liu-georgetown]
 - canonical: {first: Yang, last: Liu}
```

Comment:
This one needs to stay in! You can update the `similar` line to your ID.

Reply:
Oh I see, sorry! Just fixed it and hope it will work this time, thank you!